README.md

```diff
@@ -170,6 +170,9 @@ These may be specified with the `--format` flag.
 * Any criteria in the scorecard must be actionable. It should be possible, with help, for any project to "check all the boxes".
 * Any solution to compile a scorecard should be usable by the greater open source community to monitor upstream security.
 
+## Docker
+* The Dockerfile in the root directory utilizes [experminetal features](https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/syntax.md)
```

Comment:
nit: typo, experimental.

Do you need a specific version of docker to use this or any specific flags?

Reply:
Took care of it. Thanks